### ISSUE TITLE
proplot makes me can not import the colormaps and colors from matplotlib?

### ISSUE THREAD

**Issue by yilvzhishi** [proplot-dev/proplot#403]
When I have not installed the proplot by conda-forge in a conda env ,I can import the colormaps and colors form matplotlib,but when i install matplotlib and proplot in an env ,i find that i can't import the colormaps and colors from the matplotlib,and i create a new conda env to test in addition,i also found the question,hope the team of proplot can note and solve this issue.

When I test an env that only installed matplotlib,
![image](https://user-images.githubusercontent.com/105226929/209794762-b1eeb95a-56a8-43c9-86fc-f77f1c598455.png)
this env "pygis" only have matplotlib and not have proplot installed.
![image](https://user-images.githubusercontent.com/105226929/209795125-0865c300-e3bb-4c2e-9851-5f8e799c2978.png)
But when i change an env which have the matplotlib and proplot installed together,it can't work and raise an error.
![image](https://user-images.githubusercontent.com/105226929/209794948-728d7e78-50fd-4aa8-9e91-685e2027d934.png)
This env "test" have the matplotlib and proplot,please see:
![image](https://user-images.githubusercontent.com/105226929/209795302-a6357c25-9c83-4fa4-be5b-8f4e50775275.png)

**Comment by syrte** [comment]
Note that the matplotlib version is different in your two enviroment.
I guess it is because you just cannot import `colormaps` from matplotlib 3.4, no matter you have installed proplot or not.

**Comment by syrte** [comment]
Confirmed: matplotlib.colormaps was introduced in 3.5
https://matplotlib.org/stable/users/prev_whats_new/whats_new_3.5.0.html#colors-and-colormaps

**Comment by lukelbd** [comment]
Yep, right now the stable release 0.9.7 is only compatible with matplotlib <= 3.4. However the "dev" version has 3.5.X compatibility (you can install with `pip install git+https://github.com/proplot-dev/proplot`).
